setup.py

```diff
@@ -32,7 +32,7 @@
     ],
     keywords=["youtube", "title", "metadata", "parse", "music"],
     packages=find_packages(exclude=["contrib", "docs", "test*"]),
-    install_requires=[],
+    install_requires=["wheel","twine"],
```

Comment:
I don't believe wheel and twine are required to be a part of `install_requires`, rather they are packages used to build and publish the package. `install_requires` includes packages that are necessary for the package to run. Please confirm.
https://packaging.python.org/discussions/install-requires-vs-requirements/

Reply:
Oh I'm sorry , I forgot to remove that line😅. Yup, they aren't required.